x-pack/plugins/index_lifecycle_management/public/application/index.tsx

```diff
@@ -26,11 +28,12 @@ export const renderApp = (
   getUrlForApp: ApplicationStart['getUrlForApp'],
   breadcrumbService: BreadcrumbService,
   license: ILicense,
+  managementPageLayout: ManagementAppMountParams['managementPageLayout'],
   cloud?: CloudSetup
 ): UnmountCallback => {
   render(
     <I18nContext>
-      <KibanaContextProvider services={{ cloud, breadcrumbService, license }}>
+      <KibanaContextProvider services={{ cloud, breadcrumbService, license, managementPageLayout }}>
```

Comment:
**Step 2:** Use `KibanaContextProvider` to provide it to the app using React Context. Note that we have to update our types in the `types.ts` file to support the change to this interface.